scripts/tiny.config

```diff
@@ -524,7 +536,8 @@ CONFIG_HAVE_PCI=y
 # Generic Driver Options
 #
 # CONFIG_UEVENT_HELPER is not set
-# CONFIG_DEVTMPFS is not set
+CONFIG_DEVTMPFS=y
+CONFIG_DEVTMPFS_MOUNT=y
```

Comment:
Note: These configs are also enabled so we don't have to change the config again to boot Linux successfully in the future.